recipe-app/src/pages/newForm/components/NewForm.jsx

```diff
@@ -27,7 +27,7 @@ function NewForm() {
               <span className="number">1</span> Your Basic Info:
             </legend>
             <TextInput
-              onChange={onChangeHandler}
+              onChangeHandler={onChangeHandler}
```

Comment:
From the <TextInput> component, we're exposing a prop called onChangeHandler which maps to the onChange synthetic event on the actual input element (styled in your case). Therefore there will not be an onChange at the <TextInput> level because this component is a wrapper on the actual styled input field. Hence, we use the exposed prop - onChangeHandler to connect the onChangeHandler function in the NewForm component. I think yesterday this was called captureData function. In that case, this would be onChangeHandler={captureData} but I see you've renamed the function to onChangeHandler which is totally fine.

Reply:
I did rename it because to me, captureData makes more sense as an onSubmit handler not an onChange handler.

Reply:
Works either way, just refer to the handler function in the onChangeHandler prop